whipper/result/logger.py

```diff
@@ -148,11 +148,12 @@ def logRip(self, ripResult, epoch):
         data["EOF"] = "End of status report"
         riplog["Conclusive status report"] = data
 
+        yaml = YAML(
+            typ="unsafe",
```

Comment:
We used `yaml.RoundTripDumper` before which still seems to be the default one (`rt`).

Reply:
Changed to `rt`, can squash if acceptable.

Reply:
> Changed to `rt`, can squash if acceptable.

Yeah, that's fine